internal/install/recipe_installer.go

```diff
@@ -130,45 +140,40 @@ func (i *RecipeInstaller) Install() error {
 		}
 
 		for _, r := range recipes {
-			log.Debugf(fmt.Sprintf("Found available integration %s.", r.Name))
+			log.Debugf("Found available integration %s.", r.Name)
```

Comment:
Do we need a line feed at the end?

Reply:
We shouldn't.  The `log` statement should do that for us.  If it were a `fmt.Sprintf`, then we would.